wp-api.js

```diff
@@ -819,10 +754,21 @@
 	wp.api.collections.PostStatuses = BaseCollection.extend(
 		/** @lends PostStatuses.prototype */
 		{
-			url: WP_API_Settings.root + '/posts/statuses',
+			url: WP_API_Settings.root + '/statuses',
 
-			model: wp.api.models.PostStatus
+			model: wp.api.models.PostStatus,
 
+			parse: function( response ) {
+				var responseArray = [];
+
+				for ( var property in response ) {
+					if ( response.hasOwnProperty( property ) ) {
+						responseArray.push( response[property] );
```

Comment:
`responseArray = _.values( response )` may be more concise? `_.values` will only return own properties